src/django/api/views.py

```diff
@@ -262,3 +266,9 @@ def create(self, request):
 
         serializer = self.get_serializer(new_list)
         return Response(serializer.data)
+
+    def list(self, request):
+        organization = Organization.objects.get(name=request.user.name)
```

Comment:
The `request.user` object has an `organization` property that should be used here. It's optional, so it needs to be checked for non-None. (Fixing this would take care of #105)

Reply:
Added in 
15c572b along with a test.